action.yml

```
@@ -6,11 +6,11 @@ branding:
 outputs:
   default_branch:
     description: "The default branch name"
-    value: ${{ steps.default-branch-name.outputs.default-branch }}
 runs:
   using: 'composite'
   steps:
     - id: default-branch-name
       shell: bash
       run: |
-        echo ::set-output name=default-branch::$(curl -s https://api.github.com/repos/${{ github.repository }} -H "Authorization: bearer ${{ github.token }}" | jq -r .default_branch)
\ No newline at end of file
+        default_branch=$(curl -s https://api.github.com/repos/${{ github.repository }} -H "Authorization: bearer ${{ github.token }}" | jq -r .default_branch)
+        echo "default_branch=$default_branch" >> $GITHUB_OUTPUT
```

Thank you for your contribution!

Would it be okay to do this?

```suggestion
       curl -s https://api.github.com/repos/${{ github.repository }} -H "Authorization: bearer ${{ github.token }}" | jq -r '"default_branch=\(.default_branch)"' >> $GITHUB_OUTPUT
```

In my local machine:
```
$ echo '{"default_branch": "foo"}' | jq -r '"default_branch=\(.default_branch)"'
default_branch=foo
```